docs/pages/server-access/guides/openssh.mdx

```diff
@@ -61,10 +61,15 @@ resource on your cluster with the following command, assigning <Var name="proxy"
 and <Var name="node" /> to the name of the node that will join the cluster:
 
 ```code
-$ teleport join openssh --address <Var name="node" /> --proxy-server <Var name="proxy" /> --join-method token --token <TOKEN>
+$ sudo teleport join openssh --address <Var name="node" /> --proxy-server <Var name="proxy" /> \
```

Comment:
```suggestion
$ sudo teleport join openssh \
  --address <Var name="node" /> \
  --proxy-server <Var name="proxy" /> \
```

I would break lines for each flag, rather than having 2 flags on the first line and one flag on all following lines